static/js/click_handlers.js

```diff
@@ -596,7 +596,7 @@ $(function () {
             name: "settings-toggle",
             values: [
                 { label: "Settings", key: "settings" },
-                { label: "Administration", key: "administration" },
+                { label: "Organization", key: "administration" },
```

Comment:
We'll want to change `key` as part of this effort, but when we do that we'll need to change most references to `administration` in the codebase overall, including the hash.  So that can be a separate commit.